packages/odyssey-storybook/src/components/odyssey-mui/TextField/TextField.stories.tsx

```diff
@@ -246,6 +247,19 @@ export const Error: StoryObj<typeof TextField> = {
     errorMessage: "This field is required.",
     defaultValue: "",
   },
+  play: async ({ canvasElement, step }) => {
+    await step("has visible label", async () => {
+      const element = queryOdysseySelector({
+        canvas: within(canvasElement),
+        componentName: "TextField",
+        templateArgs: {
+          label: "Destination",
+        },
+      }).element;
+
+      expect(element).toHaveErrorMessage(/This field is required/);
```

Comment:
We can probably have this quoted instead:
```suggestion
      expect(element).toHaveErrorMessage("This field is required");
```